plone/app/querystring/interfaces.py

```diff
@@ -28,6 +28,7 @@ class IQueryField(Interface):
     description = Text(title=u"Description")
     enabled = Bool(title=u"Enabled")
     sortable = Bool(title=u"Sortable")
+    lazy = Bool(title=u"Lazy")
```

Comment:
I do not like the name "lazy" on the field. It is not very explicit. And since there is no other description than the word it could be all and nothing. 
We could call it `lazy_vocabulary`, but even that does not nail it. Wording is hard. What about `suppress_vocabulary`?
And the attribute could additional describe what it is supposed to do.

Reply:
+1 for `lazy_vocabulary`

Reply:
On wording of the field, an alternative that comes to mind from Balasz comment is the (pre)fetching of all the vocab_items.  So fetch=True or prefetch=True? 

(It would flip the Boolean though. fetch=Tue equals lazy=False)

Reply:
I like it too, but mixing it with @jensens proposal, what about `fetch_vocabulary`? Defaulting to `true`, I don't see any problem, the implementation detail might be adjusted though.

Reply:
I like `fetch_vocabulary`.